hack/e2e.sh

```diff
@@ -88,10 +88,11 @@ oc new-app -f dist/openshift/cincinnati.yaml \
 EOF
 )" \
   -p ENVIRONMENT_SECRETS="{}" \
+  -p REPLICAS="2" \
   ;
 
 # Wait for dc to rollout
-oc wait --for=condition=available --timeout=5m deploymentconfig/cincinnati || {
+oc wait --for=condition=available --timeout=10m deploymentconfig/cincinnati || {
```

Comment:
If the replicas are spawned in parallel I don't expect the initialization time to double. The two pods should have all the resources they need since they're placed on different nodes by the anti-affinity. The only limitation would be the registry (Quay in this case) and the bandwidth towards that.

Reply:
>The only limitation would be the registry (Quay in this case) and the bandwidth towards that.

Yes, it appears Quay is rate-limiting requests from the cluster so initial scrape may not succeed in 5 mins on both pods

Reply:
See https://github.com/openshift/cincinnati/pull/275#discussion_r420090801